tests/queries/0_stateless/02864_statistics_estimation.sql

```diff
@@ -49,7 +65,24 @@ SELECT replaceRegexpAll(explain, '__table1.|_UInt8|_Int8|_UInt16|_String', '')
 FROM (EXPLAIN actions=1 SELECT count(*) FROM tab WHERE c < 0/*9900*/ and b = 0/*10*/ and a = '10000'/*0*/)
 WHERE explain LIKE '%Prewhere%' OR explain LIKE '%Filter column%';
 
-ALTER TABLE tab DROP STATISTICS a, b, c;
+ALTER TABLE tab DROP STATISTICS a, b, c, d;
+
+
+SELECT 'Test statistics min_max and tdigest:';
```

Comment:
I am not too happy with the test design.

With this PR, we now have 4 different statistics types.
The user can perform certain operations on these statistics (create, drop, materialize, use for estimation, etc.).

We generally want to test as systematically as possible.

One test approach is to have a test per statistics type and duplicate tests for all possible operations in all 4 test files.

The better approach is to test along operations, i.e. have (for example) something like "02864_statistics_create_materialize_drop.sql", "02864_statistics_estimation.sql", etc. Each of these files tests all statistics types. This approach is better than the previous one as it limits duplication to more local places.

(the current approach seems a weird mix of both approaches, e.g. l. 71 says only two out of four statistics types are tested, this is not systematic testing)